snuba/manual_jobs/manifest_reader.py

```diff
@@ -1,18 +1,30 @@
 import os
-from typing import Any, Sequence
+import typing
+from typing import Sequence
 
 import simplejson
 
+from snuba.manual_jobs import JobSpec
 
-class _ManifestReader:
+
+class ManifestReader:
     @staticmethod
-    def read(filename: str) -> Sequence[Any]:
+    def read(filename: str) -> Sequence[JobSpec]:
         local_root = os.path.dirname(__file__)
         with open(os.path.join(local_root, filename)) as stream:
             contents = simplejson.loads(stream.read())
-            assert isinstance(contents, Sequence)
-            return contents
+
+            job_specs = []
+            for content in contents:
+                content_dict = dict(zip(content[::2], content[1::2]))
+                job_spec = JobSpec(
+                    job_id=typing.cast(str, content_dict.get("id")),
```

Comment:
I think it might make sense to do `typing.assert_type` here so we reject bad input (instead of just satisfying mypy) https://docs.python.org/3/library/typing.html#typing.assert_type